functionary/ui/templates/core/environment_detail.html

```diff
@@ -21,8 +21,12 @@ <h3 class="fw-semibold">
                 <a href="{% url 'ui:team-detail' pk=environment.team.id %}">{{ environment.team.name }}</a>
             </p>
         </div>
-        {% include "partials/environments/environment_management.html" %}
-        {% include "partials/variables_section.html" with parent_object=environment parent_id=environment.id %}
+        {% if user_details %}
```

Comment:
These both still need to show up so you can add users and variables to the environment